apps/i18n/common/en_us.json

```diff
@@ -1959,6 +1959,7 @@
   "specialAnnouncementHeadingPostHoc2019": "Go beyond an Hour of Code",
   "specialAnnouncementDescriptionPostHoc2019": "An hour is just the beginning. We offer comprehensive curriculum for grades K-12, professional learning programs including local workshops, and resources to help you and your school get started.",
   "sprite": "sprite",
+  "spriteLimitExceeded": "The maximum possible number of sprites in Sprite Lab is 1000. No additional sprites can be created.",
```

Comment:
It would be good to make the 1000 in this string configurable in case we decide to alter the limit later. If you include something like `{limit}` instead of the 1000 (ex. `...number of sprites in Spite Lab is {limit}. No additional...`), then when referencing this string, you can pass in an object specifying the value of "limit", ex. `msg.spriteLimitExceeded({limit: MAX_NUM_SPRITES})`

Reply:
That's really cool! Thanks - will update.